stories/Atom/Input-group/input_group.scss

```diff
@@ -0,0 +1,156 @@
+@import "../../assets/scss/mixins";
+@import "../../assets/scss/breakpoints";
+@import "../../assets/scss/variables";
+
+.label__caption,
+.help__caption,
+.error__caption {
+  @extend %paragragh-font-20;
+}
+.help__caption {
+  font-size: 14px;
+}
+.error__caption {
+  font-size: 16px;
+  color: $color-red;
+}
+
+.input-group {
+  .field__image {
+    display: inline-block;
+    position: relative;
+    width: 100%;
+    &.search {
+      &:before {
+        @extend %field__image;
+        content: "";
+        background-image: url(#{$img-path-Iconback}/Search_black.svg);
+      }
+      &.disabledstate {
+        &:before {
+          content: "";
+          background-image: url(#{$img-path-Iconback}/Searchgray.svg);
+        }
+      }
+    }
+  }
+  .field__setdate {
+    position: relative;
+  }
+  .input-type {
+    margin: 8px 0px;
+    border: 2px solid $color-black;
+    font-size: 16px;
+    width: 100%;
+    height: 46px;
+    outline: none;
+    &::placeholder {
+      color: $color-gray-500;
+    }
+    @include devicebreak(small) {
+      width: 478px;
+      padding-left: 10px;
+    }
+    &.textarea-field {
+      height: 185px;
+      @include devicebreak(small) {
+        padding-top: 15px;
+      }
+    }
+    &.search-field {
+      padding-left: 35px;
+      padding-right: 25px;
+      position: relative;
+      height: 52px;
+      @include devicebreak(small) {
+        width: 496px;
+      }
+    }
+    &.focusstate {
+      &:focus {
+        box-shadow: 0px 0px 4px #0468b1;
+      }
+    }
+    &.errorstate {
+      border: 2px solid $color-red;
+    }
+    &.disabledstate {
+      border: 2px solid $color-gray-400;
+      &::placeholder {
+        color: $color-gray-400;
+      }
+    }
+    &.date-field {
+      background: url(#{$img-path-Iconback}/calender_black.svg) no-repeat;
+      background-position: left 1px bottom 16px;
```

Comment:
@pallavikadam-srijan we can provide the background position in above background property